src/utils/common/common.ts

```diff
@@ -46,3 +46,15 @@ export const forkRefs =
       }
     });
   };
+
+/**
+ * The following interface allows data-* attributes.
+ * The basic React.HTMLAttributes interface throws errors when trying to do something like
+ * <Component wrapperProps={{ 'data-testid': 'something' }}
+ * Solution copied from: https://github.com/microsoft/TypeScript/issues/28960
+ */
+type DataAttributeKey = `data-${string}`;
+export interface HTMLAttributesIncludingDataAttributes
```

Comment:
Will this now allow all html attributes to the elements of this type regardless of the type of the element in question?

Reply:
Updated these same way as in Ilkka's PR